sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,159 @@ def __repr__(self):
+        if hasattr(self, '_mu_s'):
+            msg = (
+                f'Can\'t set attribute `coefficient_of_kinetic_friction` to '
+                f'{repr(mu_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_s = sympify(mu_s, strict=True)
+
+    @property
+    def normal_force(self):
+        return self._normal_force
+
+    @normal_force.setter
+    def normal_force(self, normal_force):
+        if hasattr(self, '_normal_force'):
+            msg = (
+                f'Can\'t set attribute `normal_force` to '
+                f'{repr(normal_force)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._normal_force = sympify(normal_force, strict=True)
+
+    @property
+    def viscous_coeffient(self):
+        """...."""
+        return self._viscous_coeffient
+
+    @viscous_coeffient.setter
+    def viscous_coeffient(self, viscous_coeffient):
+        if hasattr(self, '_viscous_coeffient'):
+            msg = (
+                f'Can\'t set attribute `viscous_coeffient` to '
+                f'{repr(viscous_coeffient)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._viscous_coeffient = sympify(viscous_coeffient, strict=True)
+
+    @property
+    def v_s(self):
+        return self._v_s
+
+    @v_s.setter
+    def v_s(self, v_s):
+        if hasattr(self, '_stribeck_velocity'):
+            msg = (
+                f'Can\'t set attribute `stribeck_velocity` to '
+                f'{repr(v_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._v_s = sympify(v_s, strict=True)
+
+    @property
+    def pathway(self):
+        """The ``Pathway`` defining the actuator's line of action."""
+        return self._pathway
+
+    @pathway.setter
+    def pathway(self, pathway):
+        if hasattr(self, '_pathway'):
+            msg = (
+                f'Can\'t set attribute `pathway` to {repr(pathway)} as it is '
+                f'immutable.'
+            )
+            raise AttributeError(msg)
+        if not isinstance(pathway, PathwayBase):
+            msg = (
+                f'Value {repr(pathway)} passed to `pathway` was of type '
+                f'{type(pathway)}, must be {PathwayBase}.'
+            )
+            raise TypeError(msg)
+        self._pathway = pathway
+
+    @property
+    def force(self):
+        v = self._pathway.extension_velocity
+        f_c = self._mu_k * self._normal_force
+        f_max = self._mu_s * self._normal_force
+        return f_c * sign(v) + (f_max - f_c) * exp(-(v / self._v_s)**2) + self._viscous_coeffient * v
+
+    @force.setter
+    def force(self, force):
+        if hasattr(self, '_force'):
+            msg = (
+                f'Can\'t set attribute `force` to {repr(force)} as it is '
+                f'immutable.'
+            )
+            raise AttributeError(msg)
+        self._force = sympify(force, strict=True)
```

Comment:
Why do you implement this `force` setter?

Reply:
Without a force setter, we fail the mypy test with the following error:
```
sympy/physics/mechanics/actuator.py:1104: error: Read-only property cannot override read-write property  [misc]
Found 1 error in 1 file (checked 1502 source files)
Error: Process completed with exit code 1.
```
So I think we need to implement a force setter.

Reply:
Can you change it to what is being used in the other classes:
```python
    @force.setter
    def force(self, force):
        raise AttributeError('Can\'t set computed attribute `force`.')
```

Reply:
Sure, I just changed it